### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by STEdirisuriya**
Changes: [Improve documentation]

Improve README.rst file's "Installing method."

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ How do I install Susi: Download, Build, Run
 Cloud|
 
 At this time, Susi AI is not provided in compiled form, you easily build it yourself. It's not difficult and done in one minute! The source code is
-hosted at https://github.com/fossasia/susi_server, you can download it and run Susi AI with:
+hosted at https://github.com/fossasia/susi_server, you can download it and run Susi AI with (Before installation you must have "Java developement kit" latest version.):
```
Comment (akhileshPandey16) [inline README.rst]: Should also mention to install latest gradle, and give links to install both JDK and GRADLE.

Comment (STEdirisuriya) [reply README.rst]: Now added.

**Comment by 0x48piraj** [comment]
@rajendraarora16 Please make valid reviews, reviews are for helping the developer and the maintainers. :+1:

File: README.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ How do I install Susi: Download, Build, Run
 Cloud|
 
 At this time, Susi AI is not provided in compiled form, you easily build it yourself. It's not difficult and done in one minute! The source code is
-hosted at https://github.com/fossasia/susi_server, you can download it and run Susi AI with:
+hosted at https://github.com/fossasia/susi_server, you can download it and run Susi AI with (Before installation you must have "Java developement kit" latest version at http://openjdk.java.net/install/ & "Gradle" latest version at https://gradle.org/install/):
```
Comment (0x48piraj) [inline README.rst]: * hosted can be Hosted
* with **Java Developement Kit or JDK** (Before ...)

**Pull request merged**